src/UnitySampleProject/Packages/manifest.json

```diff
@@ -11,6 +12,7 @@
     "com.unity.toolchain.macos-arm64-linux-x86_64": "2.0.0",
     "com.unity.toolchain.win-x86_64-linux-x86_64": "2.0.9",
     "com.unity.ugui": "1.0.0",
+    "com.unity.vectorgraphics": "2.0.0-preview.25",
```

Comment:
What's this doing in here xD I'm guessing it's one of the new dependency's dependency

Reply:
Yeah AppKit requires it 🫠